doc/guide/installation.rst

```diff
@@ -47,10 +47,7 @@ Cask is available in Homebrew_, so OS X users can just use:
 
 .. _Homebrew: http://brew.sh/
 
-Setup
-=====
-
-Add Cask to your `$PATH`:
+You may need to add Cask to your `$PATH`:
```

Comment:
Maybe we can add a note on how to check if you need to or not.  I'm not sure about OSX but on linux you can do `which cask` to check for the executable.

Reply:
How about something like:

"If the `cask` command is not available after running `brew install cask`, you may need to add Cask to your `$PATH`:"

Reply:
You know, actually, I'm not so sure about this. I'll have to look into how the brew installation works. I have a working installation and no `.cask/` in my home dir. Should this be

`export PATH="$HOME/[[[.emacsd/]]].cask/bin:$PATH"`?

Reply:
I'm not a brew user so I can't help here.